src/backend/bs_repserver

```diff
@@ -2081,6 +2081,7 @@ sub copybuild {
       }
     }
   }
+  link("$odir/history", "$dir/history") if -e "$odir/history";
```

Comment:
This looks dangerous in any case since history file just gets extended, but not recreated IIRC. so a rebuild will change both repos most likely.